src/index.ts

```diff
@@ -67,6 +67,7 @@ export default class SentryRRWeb {
     try {
       // short circuit if theres no events to replay
       if (!this.events.length) return;
+      if (this.recordOptions['errorsOnly'] && event.type !== 'error') return;
```

Comment:
I think you'll need to check that `event.type === 'transaction'` as error events do not have `type = 'error'`

Reply:
I saw that and was so puzzled why...